doc/man3/EC_GROUP_copy.pod

```diff
@@ -198,6 +210,10 @@ L<crypto(7)>, L<EC_GROUP_new(3)>,
 L<EC_POINT_new(3)>, L<EC_POINT_add(3)>, L<EC_KEY_new(3)>,
 L<EC_GFp_simple_method(3)>, L<d2i_ECPKParameters(3)>
 
+=head1 HISTORY
+
+The EC_GROUP_check_named_curve() function was added in OpenSSL 3.0.
```

Comment:
nit: this should be 3.0.0

Reply:
I have seen this notation used in multiple files. (I think the extra digit only appears after its not 0??)